extensions/amp-base-carousel/1.0/base-carousel.js

```diff
@@ -319,6 +320,19 @@ function BaseCarouselWithRef(
       }}
       tabIndex="0"
       wrapperClassName={classes.carousel}
+      contentAs={lightbox ? WithLightbox : 'div'}
```

Comment:
Do we need to graft it on top of content? Or maybe it's better to just put `WithLightbox` as a child? In the later comments I was also wondering if we could just wrap the `WithLightbox` around each slide.

Reply:
Don't necessarily need this, mostly wanted to reduce a layer of parenting in the tree. We can put `WithLightbox` as a child for more readability, but I wonder - would the carousel have different # of intermediary layers if `lightbox` is `true` (+1) versus `false`?